test/fixtures/hooks/multi/10-print_hello_kafo.rb

```diff
@@ -0,0 +1,13 @@
+boot('10-print_hello_kafo') do
```

Comment:
Syntactically you're also allowed to write
```suggestion
boot '10-print_hello_kafo' do
```

Perhaps a bit of a style thing so I'll leave it up to you.